app/templates/password/reset.hbs

```diff
@@ -0,0 +1,3 @@
+{{password/reset-password 
+  resetPassword=(route-action "resetPassword")
```

Comment:
So I believe route-action is for discoverability.  If I change to action, it errors.  I can create a controller for this action if you would like.